spec/acceptance/init_spec.rb

```diff
@@ -1,74 +1,75 @@
 # run a test task
 require 'spec_helper_acceptance'
+require 'pry'
 
 describe 'package task' do
-  include Beaker::TaskHelper::Inventory
-  include BoltSpec::Run
-
   operating_system_fact = os[:family]
   redhat_six = os[:family] == 'redhat' && os[:release].to_i == 6
 
-  def bolt_config
-    { 'modulepath' => RSpec.configuration.module_path }
+  before(:all) do
+    inventory_hash = inventory_hash_from_inventory_file
+    inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'ssh_nodes')
+    inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'winrm_nodes')
+    inventory_hash = add_feature_to_group(inventory_hash, 'puppet-agent', 'docker_nodes')
+    write_to_inventory_file(inventory_hash, 'inventory.yaml')
   end
 
-  let(:bolt_inventory) { hosts_to_inventory.merge('features' => ['puppet-agent']) }
-
   describe 'install' do
     before(:all) do
-      apply_manifest_on(default, 'package { "pry": ensure => absent, provider => "puppet_gem", }')
+      apply_manifest('package { "pry": ensure => absent, provider => "puppet_gem", }')
     end
 
     it 'installs pry', unless: (operating_system_fact == 'windows') do
-      result = run_task('package', 'default', 'action' => 'install', 'name' => 'pry', 'provider' => 'puppet_gem')
-      expect(result[0]['status']).to eq('success')
-      expect(result[0]['result']['status']).to eq('installed')
-      expect(result[0]['result']['version']).to match(%r{\d+\.\d+\.\d+})
```

Comment:
Can we preserve testing that the version is in fact updated?

Reply:
sure .done